app/controllers/members_controller.rb

```diff
@@ -1,5 +1,17 @@
 class MembersController < ApplicationController
   before_filter :authenticate_user!
+  before_filter :load_organization
+
+  # TODO: move to abstract controller for all nested resources
+  # TODO: check authorization
+  #
+  def load_organization
+    @organization = Organization.find_by_id(params[:id])
+
+    raise not_found unless @organization
+
+    @organization
+  end
```

Comment:
We could achive the same behaviour without needing this code if we rescued `ActiveRecord::RecordNotFound` in a `rescue_from` block. Furthermore, this would work for any model.